docs/how-to/horizontally-scale.md

```diff
@@ -0,0 +1,81 @@
+# How to horizontally scale
+
+A Synapse deployment can scale horizontally by running multiple Synapse processes called workers.
+While adding more units to the Synapse charm, one of them will be the Main process and others,
+the workers. This architecture has specific requirements that will be described in this document.
+
+For more details about scaling, refer to ["Scaling synapse via workers"](https://element-hq.github.io/synapse/latest/workers.html#scaling-synapse-via-workers) in the Synapse documentation.
+
+## Requirements
+
+- Synapse charm deployed and integrated with PostgreSQL charm.
+
+The tutorial ["Getting started"](https://discourse.charmhub.io/t/synapse-docs-getting-started/12737) can be used to meet this requirement.
+
+## Steps
+
+### Deploy Redis and integrate it with Synapse
+
+Run the following commands.
+
+```
+juju deploy redis-k8s --channel edge
+juju integrate synapse redis
+```
+
+Once the output of the `juju status` command shows that the units are active and idle, proceed with
+the next step.
+
+### Deploy S3-integrator and integrate it with Synapse
+
+This will enable S3 storage provider so media will be stored on a S3 bucket. Replace the
+configuration options with your specific configuration.
+
+```
+juju deploy s3-integrator s3-integrator-media --channel edge
+juju config s3-integrator-media endpoint=http://endpoint bucket=synapse-media-bucket path=/media region=us-east-1 s3-uri-style=path
```

Comment:
For my understanding, are any of the configurations in this command set to dummy values, or would this command work out of the box?

Reply:
They are dummy values. In the line before, there is a guidance regarding that "Replace the
configuration options with your specific configuration."  Similar to existing one
https://discourse.charmhub.io/t/synapse-docs-configure-smtp/13210 for example.

Reply:
Thanks for raising this. I tried to put more emphasis in that now.

Reply:
Thanks for the explanation and edit. I think it's clearer now :)